### PR TITLE
Add Restart Button

### PR DESCRIPTION
**Pull request by ashishsiot**
Add Restart Button Just Next To Start / Pause Button  Because Of No Restart Button We Want To Go Browsers Reload Button Then It Will Restart So I Add Reload Button To Make More Essay To Use.
-Thank You

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @import url(https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.5.0/css/font-awesome.css);
 @import url(https://cdnjs.cloudflare.com/ajax/libs/caiuss/1.4.1/caiuss.min.css);
 @import url(https://cdn.rawgit.com/IonicaBizau/bag.css/gh-pages/dist/bag.fluid.css);
+@import url(https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css);
```
Comment (ashishsiot) [inline css/style.css:4]: Link For CSS Of Button

File: index.html [hunk 2]
```diff
@@ -18,6 +18,12 @@
                         <i class="fa fa-pause"></i> Pause
                     </span>
                 </button>
+
+                <button  href="#" class="btn btn-info btn-lg">
+                    <span onClick="window.location.reload();">
```
Comment (ashishsiot) [inline index.html:30]: Main Reload Function Calling

File: index.html [hunk 3]
```diff
@@ -7,6 +7,13 @@
 	<script src="js/main.js"></script>
     </head>
     <body>
+        <button onclick="myFunction()">🌞||🌚</button>
+        <script>
+            function myFunction() {
+               var element = document.body;
+               element.classList.toggle("dark-mode");
+            }
+            </script>
```
Comment (ashishsiot) [inline index.html:16]: Define Button And Script for Dark Mode

**Comment by IonicaBizau** [comment]
The restart button only does a refresh currently... Dark mode would be nice, but please add the code in the script.

**Pull request closed without merging**